### ISSUE TITLE
Use genemetrics output for calling CNV?

### ISSUE THREAD

**Issue by aelhossiny** [etal/cnvkit#745]
Hi, Thanks for such a wonderful tool. Can I use `genemetrics` command output as an input for calling CNV using `call` command?

I would like to have a single CN per gene for my analysis

Thank you!

**Comment by etal** [comment]
Yes, definitely. The latest release will give you a confidence interval for the gene's copy number log2-ratio value and other statistics, too.
